CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateVideoProcessor.m

```diff
@@ -0,0 +1,410 @@
+//
+//  CRFHeartRateVideoProcessor.m
+//  CardiorespiratoryFitness
+//
+//  Copyright © 2017 Sage Bionetworks. All rights reserved.
```

Comment:
OK never mind about all the copyright date comments, these _have_ to have been copied/moved from elsewhere, right?